search-services/latest/admin/index.md

```diff
@@ -76,7 +76,9 @@ You can set different debug logging levels for Solr components using the Solr lo
 
 ### Document fingerprints
 
-Alfresco Content Services 6.2 provides support for Document Fingerprints to find related documents. Document Fingerprinting is performed by algorithms that map data, such as documents and files to shorter text strings, also known as fingerprints. This feature is exposed as a part of the Alfresco Full Text Search Query Language.
+Alfresco Content Services 6.2 and above provides support for Document Fingerprints to find related documents. Document Fingerprinting is performed by algorithms that map data, such as documents and files to shorter text strings, also known as fingerprints. This feature is exposed as a part of the Alfresco Full Text Search Query Language.
+
+> **NOTE:** Document fingersprints is disabled by default, for more see [Document Fingerprints]({% link search-services/latest/config/performance.md %}#document-fingerprints)
```

Comment:
Looks like incorrect link, should it no go to disabled info...

Reply:
@eliaporciani can you confirm if this link is correct please?
I presume I got it from you?
Cheers :-)

Reply:
done

Reply:
Link is broken: http://localhost:4000/search-services/latest/config/performance.md#disable-fingerprint

Reply:
Cheers Martin, the link says this tho [Document Fingerprints]({% link search-services/latest/config/performance.md %}#document-fingerprints)

It should say **document-fingerprints** not disable?

It works on my local. The link you're quoting says **disable-fingerprint**   .

Reply:
Yeah just checked the link on my local and all links work on that page, sorry can you check it again?
cheers.

Reply:
See https://github.com/Alfresco/docs-alfresco/pull/317/files#diff-fd15bed585d8580ea08e53bd63f7d649b6ad120c8d09586e00d6fd84fb00ea56R101

The text says: 
> **NOTE:** Document fingersprints is disabled by default, for more see [Document Fingerprints]({% link insight-engine/latest/config/performance.md %}#disable-fingerprint)

1. `#disable-fingerprint` should be `#disable-document-fingerprint` based on the sections in `insight-engine/latest/config/performance.md`. The link works but the page anchor is ignored.
2. Typo: `fingersprints`.

Reply:
All done cheers.